draft-specifications/Model.md

```diff
@@ -235,50 +284,69 @@ A struct with unique, unordered string keys and values of heterogeneous type.
 
 ## Tagged
 
-A value with a string tag.
+A [Value](#value) with a tag.
+The value may be any value.
+The tag is a string of Unicode code points excluding surrogates (U+D800-U+DFFF).
 
 > OCapN provides a small number of container types with minimal semantics.
 > Tags allow protocols to emerge from values annotated to indicate a richer
 > variety of semantics.
 > Concretely, a list might be differentiated from or hinted to be an ordered
 > list, sorted list, or a set of unique values.
 >
-> - **Guile**: currently a Guile record labeled `'(syrec ...)` (**imported**)
+> - **Guile**: currently a Guile record labeled `'(make-tagged 'tagName 10)` (**imported**)
 > - **JavaScript**: an object with the key `Symbol.for('passStyle')` and value
 >   `tagged`, with a string tagName and a single `"payload"` field carrying the tagged value.
```

Comment:
Good that the code example here and in `Selector` are similar. The text decorating that code should be similar too.

Reply:
I’ve posted a commit that regularizes the JavaScript examples in the cases where they’re objects with a passStyle.